website/docs/feature-flag-tutorials/use-cases/gradual-rollout.md

```diff
@@ -0,0 +1,160 @@
+
+You can configure your gradual rollout strategy using the following parameters in Unleash:
+
+-   **rollout percentage** (0-100%) determines the number of users you want to enable the feature flag for
+-   **stickiness** is how Unleash guarantees that the same user gets the same features every time. Stickiness is useful when you want to show a feature to only a subset of users. The same `userId` and the same rollout percentage should give predictable results. Configuration that should be supported:
+    -   **default** - Unleash chooses the first value present on the context in defined order `userId`, `sessionId`, `random`.
+    -   **userId** - guaranteed to be sticky on `userId`. If `userId` is not present, the behavior would be `false`
+    -   **sessionId** - guaranteed to be sticky on `sessionId`. If `sessionId` not present the behavior would be `false`.
+    -   **random** - no stickiness guaranteed. For every `isEnabled` call it will yield a random `true`/`false` based on the selected rollout percentage.
+    -   You can also enable **custom stickiness** by creating a context field for it to use in your gradual rollout. Learn more about [custom stickiness](/reference/stickiness#enabling-custom-stickiness).
+    -   **groupId** is used to ensure that different flags will be evaluated differently. The `groupId` defaults to feature flag name, but the user can override it to _correlate rollout_ of multiple feature flags.
+
+![](/img/tutorial-grad-rollout-form.png)
+
+There are two more advanced extensions of the gradual rollout strategy that you will see available to customize in the gradual rollout form:
+
+-   [Strategy Constraints](/reference/strategy-constraints)
+-   [Strategy Variants](/reference/strategy-variants)
+
+Constraints and variants are not required for a gradual rollout. These additional customizations can be built on top of the overall rollout strategy should you need more granular conditions for your feature beyond the rollout percentage.
+
+For gradual rollouts, _strategy constraints_ are most applicable for more granular conditions of a feature release.
+
+In the next section, we’ll explore how to apply a strategy constraint on top of a gradual rollout for more advanced use cases.
+
+## Applying Strategy Constraints
+
+When utilizing an activation strategy such as a gradual rollout, it may be necessary to further define which subset of users get access to a feature and when the rollout takes place, depending on the complexity of your use case. Unleash provides [Strategy Constraints](/reference/strategy-constraints) as a way to fine-tune conditions under which a feature flag is evaluated. Within a gradual rollout activation strategy, you can use strategy constraints to, for example:
+
+-   roll out a feature only to users in a specific region
```

Comment:
I think it would be nice to uppercase these for consistency with other lists on the page, but that might mean a rewrite for the last bullet point